From 2518465f8ed39978425a82c9a93987c3469396f1 Mon Sep 17 00:00:00 2001
From: Louis Marelle <louis.marelle@latmos.ipsl.fr>
Date: Fri, 19 Oct 2018 13:22:34 +0200
Subject: [PATCH 11/52] Correct rate constants for SAPRC99 mechanisms
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Corrects erroneous rate constants in the SAPRC99 mechanisms. For several
reactions, the rate constants were replaced with “2.20D-10” (with no
temperature dependence). In addition, there were additional weird reactions
with the same rate constant that are not present in the original SAPRC99
mechanism. This replaces the rate constants with the ones from the
original SAPRC99 publication (Carter, 2000) and removes the additional
reactions.  Reference: Carter et al., 2000: Documentation of the SAPRC-99
chemical mechanism for VOC reactivity assessment
---
 chem/KPP/mechanisms/saprc99/saprc99.eqn         | 11 ++++++-----
 .../saprc99_mosaic_4bin_vbs2.eqn                | 17 +++++------------
 .../saprc99_mosaic_8bin_vbs2_aq.eqn             | 17 +++++------------
 3 files changed, 16 insertions(+), 29 deletions(-)

diff --git a/chem/KPP/mechanisms/saprc99/saprc99.eqn b/chem/KPP/mechanisms/saprc99/saprc99.eqn
index 6bdfaf93..d4f83fa6 100755
--- a/chem/KPP/mechanisms/saprc99/saprc99.eqn
+++ b/chem/KPP/mechanisms/saprc99/saprc99.eqn
@@ -363,16 +363,17 @@
  0.659MEK + 0.259PROD2 + 
  0.012METHACRO : 2.085D-11 ;
 {214} C2H2 + O3 = 0.5OH + 1.5HO2 + 
- 1.5CO + 0.5CO2 : 2.20D-10 ;
+ 1.5CO + 0.5CO2 : ARR(2.00D-14,4400.0_dp,0.0_dp,TEMP)  ;
 {215} C3H6 + OH = 0.984RO2_R + 0.016RO2_N +
- 0.984HCHO + 0.984CCHO + 0.048XC  : 2.20D-10 ;
+ 0.984HCHO + 0.984CCHO + 0.048XC : ARR(4.85D-12,-504.0_dp,0.0_dp,TEMP) ;
 {216} C3H6 + O3 = 0.32OH + 0.06HO2 + 
  0.26C_O2 + 0.51CO + 0.135CO2 + 0.5HCHO +
- 0.5CCHO + 0.185HCOOH + 0.17CCO_OH + 0.07XC : 2.20D-10 ;
+ 0.5CCHO + 0.185HCOOH + 0.17CCO_OH + 0.07XC : ARR(5.51D-15,1878.0_dp,0.0_dp,TEMP) ;
 {217} C3H6 + NO3 = 0.949RO2_R + 0.051RO2_N + 
- 2.693XC + 1.0XN : 2.20D-10 ;
+ 2.693XC + 1.0XN :  ARR(4.59D-13,1156.0_dp,0.0_dp,TEMP)  ;
 {218} C3H6 + O3P = 0.45RCHO + 0.55MEK + 
- 0.55XC : 2.20D-10 ;{236} SESQ + OH = 0.75RO2_R + 0.25RO2_N +
+ 0.55XC : ARR(1.18D-11,324.0_dp,0.0_dp,TEMP) ; 
+ {236} SESQ + OH = 0.75RO2_R + 0.25RO2_N +
  0.5R2O2+ 0.276HCHO +
  0.474RCHO + 0.276PROD2 : ARR(1.83D-11,-449.0_dp,0.0_dp,TEMP) ;
 {237} SESQ + O3 = 0.567OH + 0.033HO2 +
diff --git a/chem/KPP/mechanisms/saprc99_mosaic_4bin_vbs2/saprc99_mosaic_4bin_vbs2.eqn b/chem/KPP/mechanisms/saprc99_mosaic_4bin_vbs2/saprc99_mosaic_4bin_vbs2.eqn
index edae9aec..51e22e89 100755
--- a/chem/KPP/mechanisms/saprc99_mosaic_4bin_vbs2/saprc99_mosaic_4bin_vbs2.eqn
+++ b/chem/KPP/mechanisms/saprc99_mosaic_4bin_vbs2/saprc99_mosaic_4bin_vbs2.eqn
@@ -370,23 +370,16 @@
  0.659MEK + 0.259PROD2 + 
  0.012METHACRO : 2.085D-11 ;
 {218} C2H2 + O3 = 0.5OH + 1.5HO2 + 
- 1.5CO + 0.5CO2 : 2.20D-10 ;
+ 1.5CO + 0.5CO2 : ARR(2.00D-14,4400.0_dp,0.0_dp,TEMP)  ;
 {219} C3H6 + OH = 0.984RO2_R + 0.016RO2_N +
- 0.984HCHO + 0.984CCHO + 0.048XC +psd1 : 2.20D-10 ;
+ 0.984HCHO + 0.984CCHO + 0.048XC +psd1 : ARR(4.85D-12,-504.0_dp,0.0_dp,TEMP) ;
 {220} C3H6 + O3 = 0.32OH + 0.06HO2 + 
  0.26C_O2 + 0.51CO + 0.135CO2 + 0.5HCHO +
- 0.5CCHO + 0.185HCOOH + 0.17CCO_OH + 0.07XC : 2.20D-10 ;
+ 0.5CCHO + 0.185HCOOH + 0.17CCO_OH + 0.07XC : ARR(5.51D-15,1878.0_dp,0.0_dp,TEMP) ;
 {221} C3H6 + NO3 = 0.949RO2_R + 0.051RO2_N + 
- 2.693XC + 1.0XN : 2.20D-10 ;
+ 2.693XC + 1.0XN :  ARR(4.59D-13,1156.0_dp,0.0_dp,TEMP)  ;
 {222} C3H6 + O3P = 0.45RCHO + 0.55MEK + 
- 0.55XC : 2.20D-10 ;
-{223} SO2 = H2SO4 : 2.20D-10 ;
-{224} HO2 = PROD : 7.0D-7 ;
-{225} SO2 = PROD : 2.20D-10 ;
-{226} H2SO4 = PROD : 2.20D-10 ;
-{227} HNO3 = PROD : 2.20D-10 ;
-{228} H2O2 = PROD : 2.20D-10 ;
-{229} CO2 = PROD : 7.0D-7 ;
+ 0.55XC : ARR(1.18D-11,324.0_dp,0.0_dp,TEMP) ; 
  {230} ALK4 + OH = ant1_c + ALK4 + OH              : Keff(5.946D-12,91.0_dp,0.0_dp,TEMP,nume,den,0.011_dp,0.022_dp) ;
  {231} ALK5 + OH = ant1_c + ALK5 + OH              : Keff(1.112D-11,52.0_dp,0.0_dp,TEMP,nume,den,0.064_dp,0.128_dp) ;
  {232} OLE1 + OH = ant1_c + OLE1 + OH              : Keff(7.095D-12,-451.0_dp,0.0_dp,TEMP,nume,den,0.0002_dp,0.0012_dp) ;
diff --git a/chem/KPP/mechanisms/saprc99_mosaic_8bin_vbs2_aq/saprc99_mosaic_8bin_vbs2_aq.eqn b/chem/KPP/mechanisms/saprc99_mosaic_8bin_vbs2_aq/saprc99_mosaic_8bin_vbs2_aq.eqn
index ce75986a..9f3906a4 100755
--- a/chem/KPP/mechanisms/saprc99_mosaic_8bin_vbs2_aq/saprc99_mosaic_8bin_vbs2_aq.eqn
+++ b/chem/KPP/mechanisms/saprc99_mosaic_8bin_vbs2_aq/saprc99_mosaic_8bin_vbs2_aq.eqn
@@ -370,23 +370,16 @@
  0.659MEK + 0.259PROD2 + 
  0.012METHACRO : 2.085D-11 ;
 {218} C2H2 + O3 = 0.5OH + 1.5HO2 + 
- 1.5CO + 0.5CO2 : 2.20D-10 ;
+ 1.5CO + 0.5CO2 : ARR(2.00D-14,4400.0_dp,0.0_dp,TEMP)  ;
 {219} C3H6 + OH = 0.984RO2_R + 0.016RO2_N +
- 0.984HCHO + 0.984CCHO + 0.048XC +psd1 : 2.20D-10 ;
+ 0.984HCHO + 0.984CCHO + 0.048XC +psd1 : ARR(4.85D-12,-504.0_dp,0.0_dp,TEMP) ;
 {220} C3H6 + O3 = 0.32OH + 0.06HO2 + 
  0.26C_O2 + 0.51CO + 0.135CO2 + 0.5HCHO +
- 0.5CCHO + 0.185HCOOH + 0.17CCO_OH + 0.07XC : 2.20D-10 ;
+ 0.5CCHO + 0.185HCOOH + 0.17CCO_OH + 0.07XC : ARR(5.51D-15,1878.0_dp,0.0_dp,TEMP) ;
 {221} C3H6 + NO3 = 0.949RO2_R + 0.051RO2_N + 
- 2.693XC + 1.0XN : 2.20D-10 ;
+ 2.693XC + 1.0XN :  ARR(4.59D-13,1156.0_dp,0.0_dp,TEMP)  ;
 {222} C3H6 + O3P = 0.45RCHO + 0.55MEK + 
- 0.55XC : 2.20D-10 ;
-{223} SO2 = H2SO4 : 2.20D-10 ;
-{224} HO2 = PROD : 7.0D-7 ;
-{225} SO2 = PROD : 2.20D-10 ;
-{226} H2SO4 = PROD : 2.20D-10 ;
-{227} HNO3 = PROD : 2.20D-10 ;
-{228} H2O2 = PROD : 2.20D-10 ;
-{229} CO2 = PROD : 7.0D-7 ;
+ 0.55XC : ARR(1.18D-11,324.0_dp,0.0_dp,TEMP) ; 
 {230} ALK4 + OH = ant1_c + ALK4 + OH                   : Keff(5.946D-12,91.0_dp,0.0_dp,TEMP,nume,den,0.011_dp,0.022_dp) ;
 {231} ALK5 + OH = ant1_c + ALK5 + OH                   : Keff(1.112D-11,52.0_dp,0.0_dp,TEMP,nume,den,0.064_dp,0.128_dp) ;
 {232} OLE1 + OH = ant1_c + OLE1 + OH                   : Keff(7.095D-12,-451.0_dp,0.0_dp,TEMP,nume,den,0.0002_dp,0.0012_dp) ;
-- 
2.31.1

